src/components/PageButton.vue

```diff
@@ -0,0 +1,52 @@
+<script setup>
+const props = defineProps({ button: { type: Object, required: true } })
+</script>
+
+<template>
+  <div class="container">
+    <button
+      class="page"
+    >
+      <aside class="icon-wrapper">
+        <component :is="button.comp" />
+      </aside>
+
+      <span class="page-text">
+        {{ button.name }}
+      </span>
+    </button>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+button {
+  background-color: transparent;
+  color: #ffff;
+  opacity: 60%;    
+  cursor: pointer;
+  border: none;
```

Comment:
Maybe you missed my comment on this before?

https://github.com/brett-yoncak/sb-intern-project/pull/4#discussion_r911322749